lib/services/config-api-service.js

```diff
@@ -30,6 +32,13 @@ function configApiServiceFactory(
         return configuration.getAll();
     };
 
+    ConfigApiService.prototype.configGetColor = function () {
+        // get the config
+        
+        var val=configuration.get("logColorEnable");
+        return {enable:(val === undefined) ? false : val};
```

Comment:
configuration.get allows you to set default value when calling it. You don't need to do it manually.
https://github.com/RackHD/on-core/blob/master/lib/services/configuration.js#L81